ui.frontend/src/view/FormField.js

```diff
@@ -77,10 +77,14 @@ class FormField {
     setActive() {
         if (!this.isActive()) {
             this.element.setAttribute(Constants.DATA_ATTRIBUTE_ACTIVE, true);
+            if(this.parentView && this.parentView._model.activeChild !== this._model) { 
+                this.parentView._model.activeChild = this._model; // updating the activeChild of the model when a field is focused in view
+            }
+            this.active = true;
         }
         if (this.parentView && this.parentView.setActive) {
```

Comment:
this recursive code should now be removed, as model needs to inform its parent and view should adapt accordingly.

Reply:
Noted.